### PR TITLE
Adds CCLOWPage model, uses STI

### PR DESCRIPTION
**Pull request by simaob**
* moves Page from to TPI to TPIPage
* updates TPI frontend to use the right pages

### CODE REVIEW

File: app/admin/cclow_page.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
+ActiveAdmin.register CCLOWPage do
```
Comment (tsubik) [inline app/admin/cclow_page.rb]: I think if this is the same, just menu is different and class name we could unified it in one file, the same as we do with Tags. Could be done later.

File: db/migrate/20191206082828_add_type_to_pages.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class AddTypeToPages < ActiveRecord::Migration[6.0]
```
Comment (tsubik) [inline db/migrate/20191206082828_add_type_to_pages.rb]: Add Page class definition

```
Page = Class.new(ActiveRecord::Base)
```

as a good practive when migrating data and using AR models.

File: app/admin/cclow_page.rb (hunk 1 above)
Comment (simaob) [reply app/admin/cclow_page.rb]: let's leave it for later

File: db/migrate/20191206082828_add_type_to_pages.rb (hunk 2 above)
Comment (simaob) [reply db/migrate/20191206082828_add_type_to_pages.rb]: Thanks, will add!

**Comment by aabdaab** [comment]
I get an error both on trying to create new TPIPage and CCLOWPage:
image

**Comment by aabdaab** [comment]
The creation and edition works 👍
However, we might need to add the type to the pages like it was in TPI. I can see on the design that some of the static pages links are in the header:
image
some in the footer:
image
and some of them in both places (like `About`, `Methodology`, etc.). So we need to add a possibility to assign one or more types to the page. As per design, the only two possible types would be `header` and `footer`.

**Comment by simaob** [comment]
You mean the menu?

**Comment by aabdaab** [comment]
yeah

**Comment by simaob** [comment]
I'll add it.

**Comment by simaob** [comment]
Enabled, added three options: Header, Footer, Both. Feel free to change that to what makes most sense.

**Comment by simaob** [comment]
In the `app/models/cclow_page.rb` file

**Pull request merged**